### ISSUE TITLE
AWX 9.0.0 cannot handle "£" in SSH credentials

### ISSUE THREAD

**Issue by jamesfreeman959** [ansible/awx#5241]
##### ISSUE TYPE
 - Bug Report

##### ENVIRONMENT
* AWX version: 9.0.0
* AWX install method: docker on linux
* Ansible version:  2.8.5
* Operating System: Ubuntu Server 18.04
* Web Browser: Chrome on Windows 10

##### STEPS TO REPRODUCE

1. Define a project, inventory, and template in the normal manner.
2. Define a machine credential to enable access to any machine to run the template against. Ensure that the password has the "£" symbol as the first character.
3. Run the template against an appropriate machine (i.e. credentials match).

##### EXPECTED RESULTS

AWX runs the Ansible playbook successfully against the remote machine.

##### ACTUAL RESULTS

AWX reports the following in the Job pane:
```
SSH password: ERROR! Unexpected Exception, this is probably a bug: 'ascii' codec can't decode byte 0xc2 in position 0: ordinal not in range(128).
```

##### ADDITIONAL INFORMATION

This used to work fine in AWX 6.1.0. We upgraded to 9.0.0 yesterday afternoon and this immediately ceased to work - it seems like support for credentials that don't contain 7-bit ASCII characters is broken.

**Comment by kdelee** [comment]
can you run `locale` on the tower task container?

Should have output something like this:
```
LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=
```

**Comment by ryanpetrello** [comment]
I've attempted to reproduce this myself with an SSH password of `££££££` and am having trouble reproducing.

<img width="747" alt="image" src="https://user-images.githubusercontent.com/214912/68218980-37afae80-ffb3-11e9-8f17-b0be94adec1d.png">

**Comment by shanemcd** [comment]
We released 9.0.1 yesterday and this might be fixed now. Can you give it a shot?

**Comment by jamesfreeman959** [comment]
I think I may have a problem with my awx_task container - I built it from scratch because I wanted to use specify an SSL cert for the awx_web container (ref: https://github.com/ansible/awx/issues/3751)

When I log into the awx_task container and run ```locale```, I see the following:

```
docker exec -it awx_task /bin/bash
bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
bash-4.4# locale
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_MESSAGES to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=en_US.UTF-8
bash-4.4#
```

Is there something I have done wrong during the build process? Perhaps something in the build environment that needs to be set first?

**Comment by wenottingham** [comment]
You probably need glibc-all-langpacks (or, at least, gllibc-minimal-langpack)

**Comment by jamesfreeman959** [comment]
Would they be installed on the host machine where the containers were built? I saw no errors during the build process, but I agree it looks like something has been left out of the container. Should this not be in the Dockerfile that builds the awx_task container, or am I misunderstanding how this gets built.

**Comment by shanemcd** [comment]
Did you ever end up trying 9.0.1? We added the glibc-langpack-en package there.

**Comment by jamesfreeman959** [comment]
Aha - no sorry - I'm on site with the client that requires this functionality next Monday so will test then and report back. Is it ok if we leave this issue open for a few more days so we can complete the test?

**Comment by jamesfreeman959** [comment]
Fixed by 9.0.1 - closing. Thank you all for your help!
